src/manager/orgsAndUsers.js

```diff
@@ -190,3 +198,19 @@ export async function changeActive(org) {
     pushToast("Successfully changed active organization");
   });
 }
+
+export async function usersInOrg(orgId) {
+  return getUsers({ orgIds: [orgId] });
+}
+
+// same as above, but exclude me
+export async function inMyOrg(organization, me) {
+  if (!organization.id) return [];
+  const users = await getUsers({ orgIds: [organization.id] });
```

Comment:
The promise returned by `getUsers` could reject, which throw an error, so there should probably be a `try/catch` around this, unless you're sure it'll be caught and handled well lower in the stack.

Reply:
Added a `.catch`. It'll just return an empty list in this case, which I think is fine, since this is only likely to throw if you've lost internet connection or the API has gone down after you've loaded the page.